### ISSUE TITLE
Support creating a conda package for RTE+RRTMGP

### ISSUE THREAD

**Issue by tcmetzger** [earth-system-radiation/pyRTE-RRTMGP#61]
Support the maintainers of the RTE+RRTMGP repository in publishing the Fortran code as a package on conda forge

**Comment by RobertPincus** [comment]
See also https://github.com/earth-system-radiation/rte-rrtmgp/issues/274

**Comment by tcmetzger** [comment]
This should be possible to do once #60 is finished

**Comment by tcmetzger** [comment]
From Robert:

> Let's go ahead and distribute only double-precision versions of the libraries on conda. Managing two sets of libraries seems complicated; also it seems hard to arrange things on the Python side to allow for both precisions.

**Comment by tcmetzger** [comment]
PR on conda forge: https://github.com/conda-forge/staged-recipes/pull/28640

**Comment by tcmetzger** [comment]
Follow-up PR in the feedstock: https://github.com/conda-forge/rte_rrtmgp-feedstock/pull/3

**Comment by makepath-alex** [comment]
Here's a link to the followup issue for adding back Windows support in conda 
https://github.com/conda-forge/rte_rrtmgp-feedstock/issues/4
